zebra-consensus/src/transaction.rs

```diff
@@ -50,7 +63,8 @@ mod tests;
 #[derive(Debug, Clone)]
 pub struct Verifier<ZS> {
     network: Network,
-    script_verifier: script::Verifier<ZS>,
+    state: Timeout<ZS>,
+    script_verifier: script::Verifier,
```

Comment:
This code *is* the future plan for NU5.

(And I don't expect any changes to the script verifier in NU6, because there's nothing proposed for transparent transfers or transaction IDs.)